setup.py

```diff
@@ -109,6 +109,15 @@ def finalize_options(self):
 
 libraries = ["gsl", "gslcblas"]
 defines = []
+extra_objects = []
+
+if os.environ.get("STATIC_GSL", False):
```

Comment:
This is a bit of a hack - it's not very nice that we're hard-coding paths here. What about having an env variable STATIC_GSL_PREFIX which, if defined, gives the prefix to where libgsl.a and libgslcblas.a can be found? We can os.path.join that to the files.